packages/homepage/gatsby-node.js

```diff
@@ -49,6 +50,69 @@ exports.createPages = async ({ graphql, actions }) => {
     toPath: '/',
   });
 
+  const blogsFromMedium = await graphql(`
+    {
+      allFeedMediumBlog {
+        edges {
+          node {
+            id
+            title
+            categories
+          }
+        }
+      }
+    }
+  `);
+
+  blogsFromMedium.data.allFeedMediumBlog.edges.forEach(edge => {
+    const slug = edge.node.title
+      .toLowerCase()
+      .replace(/[^\w ]+/g, '')
+      .replace(/ +/g, '-');
+    const id = edge.node.id;
+    if (edge.node.categories) {
```

Comment:
If no categories its a comment

Reply:
Would be good to add this as a comment (hah!) so future readers know this too.